src/common/config/decrypt.tsx

```diff
@@ -64,28 +64,40 @@ export const decryptWalletOrSigner = async (
   progressCallback: (progress: number) => void
 ): Promise<Wallet | ConnectedSigner> => {
   const provider = config.network === "local" ? new providers.JsonRpcProvider() : getDefaultProvider(config.network);
-
   if (isWalletOption(config.wallet)) {
     const decryptedWallet = await Wallet.fromEncryptedJson(config.wallet, password, progressCallback);
     const connectedWallet = await decryptedWallet.connect(provider);
     return connectedWallet;
   } else {
-    const kmsCredentials = {
-      accessKeyId: config.wallet.accessKeyId, // credentials for your IAM user with KMS access
-      secretAccessKey: password, // credentials for your IAM user with KMS access
-      region: config.wallet.region,
-      keyId: config.wallet.kmsKeyId,
-    };
+    switch (config.wallet.type) {
+      case "ENCRYPTED_JSON":
+        return (await Wallet.fromEncryptedJson(config.wallet.encryptedJson, password, progressCallback)).connect(
+          provider
+        );
+      case "AWS_KMS":
+        return decryptAWSKMS(config.wallet, password, provider);
+      default:
+        throw new Error("Wallet type not supported.");
+    }
+  }
+};
+
+const decryptAWSKMS = async (wallet: AwsKmwSignerOption, password: string, provider: providers.BaseProvider) => {
+  const kmsCredentials = {
```

Comment:
```suggestion
const decryptAwsKms = async (wallet: AwsKmwSignerOption, password: string, provider: providers.BaseProvider) => {
  const kmsCredentials = {
```

since the rest already using the small case

Reply:
Okay.done